### DISCUSSION TITLE
Coqui Website Model Usage

### DISCUSSION THREAD

**Discussion by Malfaro43** [coqui-ai/TTS#2168] [General Q&A]
Hello everyone,

I've seen that you are able to synthesize voice samples on Coqui's website with astounding quality. Is it possible to access this upscaled YourTTS with python, assuming you are subscribed to the creator plan? If not, when will this become available?

Thanks!

**Comment by erogol** [comment]
There is no exposed API yet. Can you send an email to info@coqui.ai with your use case? We have an experimental API that we can provide upon request.
